src/functions/pollUsers/framework/repo/mysql/examiner-repository.ts

```diff
@@ -23,17 +24,23 @@ export const getActiveExaminers = async (): Promise<string[]> => {
   const queryResult = await query(
     connection,
     `
-    select e.individual_id, e.staff_number
+    select e.individual_id, e.staff_number, eg.test_centre_manager_ind
     from EXAMINER e
         left join EXAMINER_STATUS es on es.individual_id = e.individual_id
+        left join EXAMINER_GRADE eg on eg.examiner_grade_code = e.examiner_grade_code
     where IFNULL(e.grade_code, 'ZZZ') <> 'DELE'
     and IFNULL(es.end_date, '4000-01-01') >= ?
     `,
     [moment().format('YYYY-MM-DD')],
   );
 
   // @ts-ignore
```

Comment:
Can the `ts-ignore` be removed?

Reply:
I didn't add it, so hesitant at removing it.

Reply:
Try removing it, if the compiler will allow it (which I think it should now) then get rid of it! 🗑😁

Reply:
Done